### ISSUE TITLE
Uncaught TypeError: Luminous is not a constructor

### ISSUE THREAD

**Issue by jmagnusson** [imgix/luminous#29]
I got `drift` to work after following @paulstraw's [temp fix for Browserify](https://github.com/imgix/drift/issues/39#issuecomment-240838422). But with `luminous` I haven't been as successful. I get `Uncaught TypeError: Luminous is not a constructor` when running the "browserified" version of the following code in the browser.

```javascript
var Luminous = require('luminous-lightbox/dist/Luminous.min.js');
new Luminous(document.querySelector('#myimg'));
```

Where `Luminous` is an empty `Object {}`.

Using Browserify @ 13.1.1.

**Comment by paulstraw** [comment]
@jmagnusson, I think this was happening because `noParse` needs the full path. Can you try it with the value of `require.resolve('luminous-lightbox')`?

**Comment by marcohamersma** [comment]
I'm having the same issue, although I'm using the CLI like: (`$ browserify js/main.js --noparse=$PWD/js/luminous.js`)

@paulstraw, I think you're describing is the issue where browserify seems to ignore the `noParse` option if the path is't absolute. I had that issue initially, only to get exposed to this one. I'm not sure this is is related, what do you think?

**Comment by paulstraw** [comment]
You're totally right, I was misunderstanding the original error. I've submitted a PR that reworks how Luminous interacts with build tools like Browserify. You shouldn't need to do the `noparse` stuff anymore, and should be able to `require('luminous-lightbox')` directly, as expected.

In my tests, this is working great. I've also got a friend who's using Luminous in one of his projects giving it a go. It would be awesome if either of you could try #35 before I merge it in, as a sanity check. There's more details in [that branch's readme](https://github.com/imgix/luminous/blob/1917b2a8b09e00b152af13643fa358a652b0ddaa/README.md).

**Comment by marcohamersma** [comment]
Hmm, I'm getting `Cannot find module 'babelify'…`. If I then install babelify it throws similar errors for Babel presets.

I'm not using any babel-related things on my current project. Not sure why browserify is trying to  process the module. Either with or without the `noparse` option.  Does this make any sense to you?

**Comment by paulstraw** [comment]
Ah, I forgot to mention that these changes now no longer require `noparse`. Can you give it a go without?

**Comment by paulstraw** [comment]
Turns out, I had to bump `babelify`, `babel-preset-es2015`, and `babel-preset-stage-0` up to `dependencies` in Luminous' package.json. @marcohamersma, if you `npm update luminous-lightbox` while it's pointed at that branch, this should work now. Let me know!

**Comment by marcohamersma** [comment]
Right, that's it! works like a charm 👍

**Comment by paulstraw** [comment]
This has been fixed and released as 1.0.1: https://github.com/imgix/luminous/releases/tag/1.0.1
